test/alternator/test_batch.py

```diff
@@ -8,8 +8,12 @@
 
 import pytest
 import random
+import sys
 from botocore.exceptions import ClientError
-from util import random_string, full_scan, full_query, multiset, scylla_inject_error
+from alternator_util import random_string, full_scan, full_query, multiset
+
+sys.path.insert(1, sys.path[0] + '/../cql-pytest')
+from util import scylla_inject_error
```

Comment:
yeah, that makes sense until we have a more formal way to provide a common library to all our scylla pytests